### PR TITLE
ft: Add simple types for base styles

### PR DESCRIPTION
**Pull request by jtrussell**
Nothing too crazy, mostly will just help get autocomplete working in your editor.

### CODE REVIEW

**Comment by Jdstuart13** [comment]
Just so I know that I am understanding the goal here, this would be to get autocomplete working for the ivh-base-components?

**Comment by jtrussell** [comment]
This PR just covers ivh-base-styles. E.g.:

image

Though the components are coming too :)

File: packages/ivh-base-style/lib/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+  export const colorDanger: string
+
+  export const colorToDo: string
+  export const colorInProgress: string
+  export const colorDone: string
+
+  /**
+   * Utilitarian colors
+   */
+  export const colorText: string
```
Comment (evsheffield) [inline packages/ivh-base-style/lib/index.d.ts:104]: We added some new variables for "dark text" for use in the modals that we could add types for as well.

- https://github.com/iVantage/ivh-base/blob/master/packages/ivh-base-style/lib/index.js#L22
- https://github.com/iVantage/ivh-base/blob/master/packages/ivh-base-style/lib/index.js#L110

Comment (jtrussell) [reply packages/ivh-base-style/lib/index.d.ts:104]: Ah good - catch!

**Pull request merged**